tests/test_fe_handlers.py

```diff
@@ -100,3 +100,72 @@ def test_usertokens(session, users, http_client, base_url):
             headers={'X-Grouper-User': user.username})
     assert resp.code == 200
     assert "Disabled token: myFoobarToken" in resp.body
+
+@pytest.mark.gen_test
+def test_request_emails(graph, groups, permissions, session, standard_graph, users, base_url,
+                        http_client):
+    tech = groups["tech-ops"]
```

Comment:
Emails are sent via fe code, which the requests tests don't touch (because they use the test.util functions which in turn directly use the Group methods)